### ISSUE TITLE
Repeatedly logging confusion matrix results in overlapping numbers

### ISSUE THREAD

**Issue by sooheon** [wandb/client#1155]
`wandb --version && python --version && uname`

* Weights and Biases version: wandb, version 0.9.3
* Python version: Python 3.7.6
* Operating System: Linux

### Description

What I want: log confusion matrix for multiclass predictions on the validation set after every epoch, and have them logged separately so I can flip through them

What happened: The numbers come up overlapped on a single plot:

<img width="1037" alt="Screen Shot 2020-07-13 at 8 22 03 PM" src="https://user-images.githubusercontent.com/302945/87299187-85b78180-c546-11ea-9e84-91af45154e7c.png">

### What I Did

I'm using it through the pytorch-lightning API. The `validation_epoch_end` method returns 

```
{ 'log': {'val_loss': mean_losses,
             'confusion_matrix': confusion_matrix(y, pred, atom_syms)}}
```

and the 'log' key is simply logged via the wandb run object. `confusion_matrix` is from wandb.sklearn.

**Comment by sooheon** [comment]
Found a timely post: https://app.wandb.ai/mathisfederico/wandb_features/reports/Visualizing-Confusion-Matrices-with-W%26B--VmlldzoxMzE5ODk

Here the author uses a plotly plot rather than the built in confusion matrix.

**Comment by lavanyashukla** [comment]
Hi Sooheon, logging confusion matrices across steps isn't support natively yet. The plotly way recommended by [Mathïs](https://app.wandb.ai/mathisfederico/wandb_features/reports/Visualizing-Confusion-Matrices-with-W%26B--VmlldzoxMzE5ODk) is the recommended way to log them across steps right now.

**Comment by sooheon** [comment]
Sounds good.

**Comment by LucaMalagutti** [comment]
> Hi Sooheon, logging confusion matrices across steps isn't support natively yet. The plotly way recommended by [Mathïs](https://app.wandb.ai/mathisfederico/wandb_features/reports/Visualizing-Confusion-Matrices-with-W%26B--VmlldzoxMzE5ODk) is the recommended way to log them across steps right now.

Are there updates on this? I'm also looking to log a different confusion matrix for every epoch/step
